api/Window.json

```diff
@@ -509,6 +509,54 @@
           }
         }
       },
+      "find": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/find",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": null
```

Comment:
according to the bugzilla ticket mentioned in the issue, this should be `false`, not `null`

Reply:
There's only an MDN link on the issue, and neither of the bugs on that page mention Edge or IE (at least, not that I can see).  I also wouldn't really count on bug reports for Firefox having fully accurate data for IE.

Can you link to the Bugzilla ticket you're referring to?

Reply:
hmm, I was referring to https://bugzilla.mozilla.org/show_bug.cgi?id=672395
The link is in the MDM page indeed.

Reply:
Ah, no wonder I didn't find it the first time around -- they're using the abbreviation for IE (which I didn't look for).  Can now confirm that both IE and Edge do not support this.